.github/workflows

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
[REQUIRED] Please make sure that There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up Linters) and make sure that the test.md file is actually ignored by git and is not present in your GitHub repository.